CHANGELOG.ios.md

```diff
@@ -4,6 +4,17 @@
 - [iOS 2.0 SDK](http://aka.ms/gc6fex)
 - [iOS 1.2.4 SDK](http://aka.ms/kymw2g)
 
+### Version 2.2.0
+- Made Id property of records case-insensitive [328aadf](https://github.com/Azure/azure-mobile-services/commit/328aadf)
+- Added support for returning the underlying NSOperation on performing push, pull and purge operations [7c37f60](https://github.com/Azure/azure-mobile-services/commit/7c37f60)
+- Added support for configuring the page size of a pull operation [0c31aa3](https://github.com/Azure/azure-mobile-services/commit/0c31aa3)
+- Added support for NSUUID in PredicateTranslator [24c5a61](https://github.com/Azure/azure-mobile-services/commit/24c5a61)
+- Added support for turning off data store handling of table operations [8568d22](https://github.com/Azure/azure-mobile-services/commit/8568d22)
```

Comment:
This is more a beta flag, not sure I'd call it out in the release notes. I don't want most devs using it at this point.